src/locales/pl.ts

```diff
@@ -90,34 +90,34 @@ export default {
     peter_and_paul_apostles: 'Świętych Apostołów Piotra i Pawła',
     presentation_of_the_lord: 'Ofiarowanie Pańskie',
     transfiguration: 'Przemienienie Pańskie',
-    trinity_sunday: 'Niedziela Najświętszej Trójcy',
+    trinity_sunday: 'Najświętszej Trójcy',
   },
   sanctoral: {
     adalbert_of_prague_bishop: 'Św. Wojciecha, biskupa i męczennika',
     adalbert_of_prague_bishop_patron_of_poland: 'Św. Wojciecha, biskupa i męczennika, głównego patrona Polski',
     agatha_of_sicily_virgin: 'Św. Agaty, dziewicy i męczennicy',
-    agnes_cao_guiying_martyr: 'Św. Agnieszki Cao Guiying, Męczennicy',
+    agnes_cao_guiying_martyr: 'Św. Agnieszki Cao Guiying, męczennicy',
     agnes_of_rome_virgin: 'Św. Agnieszki, dziewicy i męczennicy',
-    alberic_crescitelli_priest: 'Św. Alberyka Crescitellego, Kapłana i Męczennika',
+    alberic_crescitelli_priest: 'Św. Alberyka Crescitellego, prezbitera i męczennika',
     albert_chmielowski_religious: 'Św. Brata Alberta Chmielowskiego, zakonnika',
     albert_the_great_bishop: 'Św. Alberta Wielkiego, biskupa i doktora Kościoła',
     all_souls: 'Wspomnienie wszystkich wiernych zmarłych',
     aloysius_gonzaga_religious: 'Św. Alojzego Gonzagi, zakonnika',
     aloysius_versiglia_bishop_and_callistus_caravario_priest_martyrs:
-      'Świętych Alojzego Versiglii, Biskupa i Kaliksta Caravaria, Kapłana',
+      'Świętych Alojzego Versiglii, biskupa i Kaliksta Caravaria, prezbitera',
     alphonsus_liguori_bishop: 'Św. Alfonsa Marii Liguoriego, biskupa i doktora Kościoła',
     ambrose_of_milan_bishop: 'Św. Ambrożego, biskupa i doktora Kościoła',
     andrew_apostle: 'Św. Andrzeja, Apostoła',
-    andrew_apostle_patron_of_scotland: 'Św. Andrzeja, Apostoła i patrona Szkocji',
-    andrew_bobola_priest: 'Św. Andrzeja Boboli, prezbitera i męczennika',
+    andrew_apostle_patron_of_scotland: 'Św. Andrzeja, Apostoła, patrona Szkocji',
+    andrew_bobola_priest: 'Św. Andrzeja Boboli, prezbitera i męczennika, patrona Polski',
```

Comment:
@jarosz, could you add another key for this celebration? The original key name (without the `patrona Polski`) is also used in Lithuania and Ukraine.

Also, try to localise it other other locales where the ‘base’ celebration (the one without the patronage title) is already present (it should be just a simple copy-paste work).